src/richie/apps/courses/models.py

```diff
@@ -190,6 +195,52 @@ def save(self, *args, **kwargs):
             self.organizations.add(self.organization_main)
 
 
+class CourseRun(BasePageExtension):
+    """
+    The course run represents and records the occurence of a course between a start
+    and an end date.
+    """
+
+    course = models.ForeignKey(
+        "Course",
+        related_name="courserun",
```

Comment:
same convention, and snakecase, related_name should ne called `course_runs`

Also, not sure this relation is necessary, shall we not do just like for organizations: Course.course_runs and Course.main_course_run ?
If not, I still think this relational schema is probably wrong 🤔  WDYT ?

Reply:
I am aiming for the user to choose within the course only among the course runs that were created for it. Is it possible?

I like your idea of having a course.main_course_run as well as my existing course.course_runs. It would make the choice of course run to show on the course card much easier later on.